[Bug]: Starting from version 10.2.20260501, the Remote WebDriver can no longer be instantiated

Till version `10.2.20260424`. the session was instantiated normally against remote server but starting from `10.2.20260501`different exception is coming up:

- Version **10.2.20260501**:
`Driver Factory Action "setRemoteDriverInstance" failed. With the following test data "Failed to create Remote WebDriver instance.". Root cause: "org.openqa.selenium.SessionNotCreatedException: Could not start a new session. Response code 500. Message: internal error "
`
- Version **10.2.20260505** gives:
```
Caused by: java.lang.NoClassDefFoundError: com/epam/reportportal/testng/ITestNGService
	at com.shaft.driver.DriverFactory.reloadProperties(DriverFactory.java:79)
	at com.shaft.driver.DriverFactory.readLastMinuteUpdatedProperties(DriverFactory.java:57)
	at com.shaft.driver.DriverFactory.getHelper(DriverFactory.java:175)
	at com.shaft.driver.SHAFT$GUI$WebDriver.<init>(SHAFT.java:120)

```
- Version **10.2.20260523** : `java.lang.RuntimeException: Driver Factory Action "setRemoteDriverInstance" failed. With the following test data "Failed to create Remote WebDriver instance.". Root cause: "org.openqa.selenium.SessionNotCreatedException: Could not start a new session. Response code 500. Message: internal error "
	at com.shaft.tools.io.internal.FailureReporter.fail(FailureReporter.java:34)
	at com.shaft.driver.internal.DriverFactory.DriverFactoryHelper.failAction(DriverFactoryHelper.java:133)
	at com.shaft.driver.internal.DriverFactory.DriverFactoryHelper.setRemoteDriverInstance(DriverFactoryHelper.java:682)
	at com.shaft.driver.internal.DriverFactory.DriverFactoryHelper.configureRemoteDriverInstance(DriverFactoryHelper.java:690)
	at com.shaft.driver.internal.DriverFactory.DriverFactoryHelper.createNewRemoteDriverInstance(DriverFactoryHelper.java:614)
	at com.shaft.driver.internal.DriverFactory.DriverFactoryHelper.initializeDriver(DriverFactoryHelper.java:860)
	at com.shaft.driver.DriverFactory.getHelper(DriverFactory.java:182)
	at com.shaft.driver.SHAFT$GUI$WebDriver.<init>(SHAFT.java:120) `

--- a/shaft-engine/src/main/java/com/shaft/properties/internal/Web.java
+++ b/shaft-engine/src/main/java/com/shaft/properties/internal/Web.java
@@ -27,7 +27,7 @@ public interface Web extends EngineProperties<Web> {
     String targetBrowserName();
 
     @Key("forceBrowserDownload")
-    @DefaultValue("true")
+    @DefaultValue("false")
     boolean forceBrowserDownload();
 
     @Key("headlessExecution")

